### ISSUE TITLE
Administration/forms GUI has badly arranged controls

### ISSUE THREAD

**Issue by Trodrige** [LibreHealthIO/LibreEHR#748]
The page has poorly arranged controls and choice of colors. These need to be adjusted.

**Comment by aethelwulffe** [comment]
Yep.  It is a total computer-nerd designed "gotta hold you mouth right to get it to work" kind of interface.
There are tons of issues that could be linked to this one.
